master/getting-started/bare-metal/bare-metal.md

```diff
@@ -31,21 +32,16 @@ However, for host endpoints, Calico is more lenient; it only polices
 traffic to/from interfaces that it's been explicitly told about. Traffic
 to/from other interfaces is left alone.
 
-As of Calico v2.1.0, Calico applies host endpoint security policy both to traffic
-that is terminated locally, and to traffic that is forwarded between host
-endpoints.  Previously, policy was only applied to traffic that was terminated
-locally.  The change allows Calico to be used to secure a NAT gateway or router.
-Calico supports selector-based policy as normal when running on a gateway or router
-allowing for rich, dynamic security policy based on the labels attached to your
-workloads.
-
-> **Note**: If you have a host with workloads on it then traffic that is forwarded to
-> workloads bypasses the policy applied to host endpoints. If that weren't the
-> case, the host endpoint policy would need to be very broad to allow all
-> traffic destined for any possible workload.
+Host endpoint policy allows Calico to be used to secure a NAT gateway or router. Calico supports selector-based policy as normal when running on a gateway or router allowing for rich, dynamic security policy based on the labels attached to your
```

Comment:
This sounds very nice but it is a little vague. I would suggest tweaking it a little bit. 

- Instead of the passive voice in the first sentence, you could make it more active, interesting, and specific by changing it to the active voice, such as the following: "You can use host endpoint policy to secure NAT gateways and routers." 

- I must say, I am not sure just what "host endpoint policy" is, although it does sound official. Perhaps you could also link them to a place in the documentation that explains what this is. I am not sure, but I think that our only documentation about this is here: /reference/calicoctl/resources/policy. If there is a better place, feel free to use that instead.

- Also, I think the "as normal" in the second sentence could be a little confusing and overly complex, so I would suggest you could also change the second sentence to something like: "Attaching labels to your workloads allows you to write [selector expressions](/reference/calicoctl/resources/policy#selector) to control the endpoints that policies get applied to." 

- I must say, I am not sure how to apply a label to a host endpoint workload. Is that explained anywhere? I can only see some stuff in the Kubernetes getting started. I think it might be something worth some discussion here if it is not explained elsewhere.

Reply:
Agree.  The second sentence is more about running calico on a NAT gateway so no need to specify workload.